docs/getting_started.md

```diff
@@ -51,6 +51,8 @@ druid-operator$ kubectl apply -f examples/tiny-cluster-zk.yaml
 druid-operator$ kubectl apply -f examples/tiny-cluster.yaml
 ```
 
+Note that above tiny-cluster only works on a single node kubernetes cluster(e.g. typical k8s cluster setup for dev using kind or minikube) as it uses local disk as "deep storage".
```

Comment:
This comment also belongs at the top of examples/tiny-cluster.yaml where it will be seen

Reply:
yeah, makes sense .. I will add that comment in file also.